app/src/main/java/org/p2p/wallet/history/ui/history/HistoryContract.kt

```diff
@@ -3,15 +3,16 @@ package org.p2p.wallet.history.ui.history
 import org.p2p.wallet.common.mvp.MvpPresenter
 import org.p2p.wallet.common.mvp.MvpView
 import org.p2p.wallet.common.ui.recycler.PagingState
+import org.p2p.wallet.history.model.HistoryItem
 import org.p2p.wallet.history.model.HistoryTransaction
 
 interface HistoryContract {
     interface View : MvpView {
         fun showPagingState(state: PagingState)
-        fun showHistory(items: List<HistoryTransaction>)
         fun openTransactionDetailsScreen(transaction: HistoryTransaction)
         fun showRefreshing(isRefreshing: Boolean)
         fun scrollToTop()
+        fun showHistory(items: List<HistoryTransaction>, moonpayItems: List<HistoryItem.MoonpayTransactionItem>)
```

Comment:
Why you didn't unify MoonpayTransaction to `HistoryTransaction`?

Reply:
HistoryTransaction has blockchain data inside, we don't have that in moonpay

Reply:
I think we can remove default parameters from the sealed class and blockchain data will not be required then